src/gobprepare/importers/dump_importer.py

```diff
@@ -0,0 +1,107 @@
+import contextlib
+from io import StringIO
+import tempfile
+import gzip
+import re
+import tempfile
+from typing import List, Iterator, Optional
+
+from gobprepare.importers.typing import ReadConfig, SqlDumpImporterConfig
+from gobconfig.datastore.config import get_datastore_config
+from gobcore.datastore.factory import DatastoreFactory
+from gobcore.datastore.objectstore import ObjectDatastore
+from gobcore.datastore.sql import SqlDatastore
+from gobcore.logging.logger import logger
+
+from gobcore.exceptions import GOBException
+
+@contextlib.contextmanager
+def _load_from_objectstore(datastore: ObjectDatastore) -> Iterator[str]:
+    datastore.connect()
+
+    obj_info = next(datastore.query(None), None)
```

Comment:
```
try:
    obj_info = next(datastore.query(None)
except StopIteration:
    raise GOBException('.......')
```

Reply:
Done.
Deze staat ook in csv_importer.py. Zal ik het ook de csv_importer aanpassen? of moet het in een andere aparte ticket?

Reply:
Als csv_importer een simpele aanpassing is kun je die prima meenemen hier. Niet nodig een apart ticket aan te maken.

Kijk ook even naar Roel z'n opmerking hierop. Dat tweede argument None dat je naar next meegeeft voorkomt dat die StopIteration raised wordt.

Reply:
De opmerking van Roel is verwerkt.
Ik heb de csv_importer niet aangepast. Het wel een kleine aanpassing  want ik moet ook lokaal goed kunnen testen. Maar aangezien de issues die ik heb met lokaal testen durf ik niet te doen, en het zou meer tijd kosten.
Misschien en refactoring ticket aanmaken voor wanneer we tijd hebben om het op te pakken.